org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -166,6 +166,44 @@ GithubOidcSageBionetworksSynapse:
       - !Ref SynapseProdAccount
     Region: us-east-1
 
+GithubOidcSageBionetworksSynapseBuild:
+  Type: update-stacks
+  DependsOn: GithubOidcSageBionetworks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.10.4/templates/IAM/github-oidc-provider.j2
+  StackName: !Sub ${resourcePrefix}-${appName}-sage-bionetworks-synapse-build
+  Parameters:
+    ProviderArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-ProviderArn' ]
+    ProviderRoleName: !Sub ${resourcePrefix}-${appName}-sage-bionetworks-synapse-build
+    ManagedPolicyArns:
+      - "arn:aws:iam::aws:policy/AdministratorAccess"
+  TemplatingContext:
+    Repositories:
+      - owner: "Sage-Bionetworks"
+        name: "Synapse-Repository-Services"
+        branches: ["*"]
+      - owner: "brucehoff"
```

Comment:
Do developers need to deploy from their forks? Could each developer have a branch on the origin repo instead?

Also, I don't know much about the platform build process, but if these are all deploying to the same EC2 then I worry about developers breaking the build with parallel deploys, or one failed build blocking all other developers.